### PR TITLE
[EAP7-1799] Add proposal for otel QS

### PR DESCRIPTION
**Pull request by jasondlee**
https://issues.redhat.com/browse/EAP7-1799

Add proposal

### CODE REVIEW

File: quickstarts/EAP7-1799-add-opentelemetry-quickstart.adoc [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+////
+Draft verbiage for up to a few sentences on the feature for inclusion in the
+Release Note blog article for the release that first includes this feature. 
+Example article: http://wildfly.org/news/2018/08/30/WildFly14-Final-Released/.
+This content will be edited, so there is no need to make it perfect or discuss
+what release it appears in.  "See Overview" is acceptable if the overview is
+suitable. For simple features best covered as an item in a bullet-point list 
+of features containing a few words on each, use "Bullet point: <The few words>" 
+////
+A new jaxrs-jwt quickstart that demonstrates a JAX-RS secured application using
```
Comment (kstekovi) [inline quickstarts/EAP7-1799-add-opentelemetry-quickstart.adoc:110]: Is this sentence related to a OpenTelemetry QS?

Comment (jasondlee) [reply quickstarts/EAP7-1799-add-opentelemetry-quickstart.adoc]: Copy and paste error. Just missed this section. Changes pushed.

**Comment by bstansberry** [comment]
@jasondlee I'm closing this instead of merging as this is not about WildFly.
